src/store/reducers/movieDetailsReducer.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+    movieDetail: {},
+  };
+  
+  const movieDetailsReducer = (state = initialState, action) => {
+    switch (action.type) {
+      case "GET_MOVIE_DETAILS":
```

Comment:
As mentioned in a comment for "getMovieDetails", this should be stored under "src/store/actionTypes.js"